fastparquet/writer.py

```diff
@@ -737,11 +788,19 @@ def write(filename, data, row_group_offsets=50000000,
         resolution; in "int96" mode, they are written as 12-byte blocks, with
         the first 8 bytes as ns within the day, the next 4 bytes the julian day.
         'int96' mode is included only for compatibility.
+    quantization_level: int or None
+        If not None, then quantization_level floating point values in this many units of the
+        median absolute deviation of the column. Only works for the 'simple' file
+        scheme.
 
     Examples
     --------
     >>> fastparquet.write('myfile.parquet', df)  # doctest: +SKIP
     """
+
+    global COLUMN_MADS
+    COLUMN_MADS = {}
```

Comment:
Should it be possible for the user to provide their own scaling, so that there's no need to compute it? That would be a lot of parameters floating around...